### ISSUE TITLE
<Basic Ruby>: <Add clarification on debugging with rdbg>

### ISSUE THREAD

**Issue by Kelechi125** [TheOdinProject/curriculum#28519]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

![Image 7-28-24 at 7 01 AM](https://github.com/user-attachments/assets/2cd70d79-9462-4571-ae98-200a1f0f0ec5)

On Step 4, it needs to be clarified that you must have a folder open and a Ruby file selected in order for this to work. 

Suggested edits for the first few sentences: 

"Now that you’re familiar with the basics, we’re going to have some fun with VSCode! Check the [VSCode rdbg Ruby Debugger documentation](https://github.com/ruby/vscode-rdbg). Generate the configuration inside your VSCode by going to Run and Debug, opening a folder, and clicking on create a launch.json file then picking Ruby (rdbg). **You must have a folder open in order for this to work.**"

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-debugging

### (Optional) Discord Name

Kelechi

### (Optional) Additional Comments

When I was going through this lesson a few weeks ago, after I first tried adding a configuration file, my Run and Debug screen changed, so it's not looking how it originally looked, so the steps for me to add the configuration file has become even a little more different because when I go to Run and Debug, it shows me the different sections of "Variables", "Watch", "Call Stack", etc. For me personally to add a new config file when here, I have to go up top and next to where the little green arrow for debugging where it might say something along the lines of "Debug current file with ..." and select "Add Configuration", and then it'll open up the launch.json file, and then I'd have to click on "Add Configuration" that shows up directly in the file.

With all of this being said, I didn't completely write my suggestion fully based on how it appears for me since I can't seem to close the Run and Debug area.

**Comment by rlmoser99** [comment]
@Kelechi125 I've been looking into this lesson and I agree that it can be confusing about needing a folder open at this point. So thank you so much for making this issue.

I think this assignment needs a deeper re-wording then just your suggestion, because the following point mentions creating a `script.rb` file that should have been created earlier in the lesson. So I think we should start with that file already being open in step 3 (and being part of a project folder).

I also wonder about breaking up step 4 into multiple points, because it is a bit hard to follow in paragraph form.

I realize that this issue is quite old now, but I wanted to ask if you wanted to work on updating this, or if you would like for someone else to pick it up.

**Comment by rlmoser99** [comment]
I'm assigning this to myself and will get a PR in the next few days to address this concern.
